### ISSUE TITLE
Browser darkmode color contrast may need adjustments

### ISSUE THREAD

**Issue by d33bs** [numfocus/MOSS#49]
In testing the work found in #47 I found that the frontend when viewed through a browser with darkmode enabled has low color contrast which might need to be adjusted. The color contrast can be important for ease of use as well as web accessibility (for those who may not be able to distinguish the text without a certain amount of color contrast). Here's a helpful link which shows the font color with background color as failing WCAG specifications: https://webaim.org/resources/contrastchecker/?fcolor=DEE2E6&bcolor=FFFFFF

<img width="927" height="574" alt="Image" src="https://github.com/user-attachments/assets/c5c4d144-dfbe-41dc-a603-bd51750ae33a" />

**Comment by ara2105** [comment]
Hii @d33bs 

I can take this up. My approach would be
Identify the root cause behind the issue (likely related to a missing dependency or rendering logic. Then, implement a fix by refactoring component. 
Test across scenarios to ensure stability.

I am ready with my solution and will start working once you give me the green flag. :)
